extras/rya.export/export.client/src/main/java/org/apache/rya/export/client/conf/MergeConfigurationCLI.java

```diff
@@ -149,7 +145,12 @@ public static MergeToolConfiguration createConfigurationFromFile(final File conf
             throw new MergeConfigurationException("Failed to create a config based on the provided configuration.", JAXBe);
         }
     }
-
+    /**
+     * returns the time option and makes it required.
+     * This is for API and is not used internally as of v3.2.11
+     * @return
```

Comment:
Add description to return and throws.  Make it clear that it returns a time or throws an exception when no time is configured because time is required.